src/client/actions/MessageReactionRemoveAll.js

```diff
@@ -24,6 +27,7 @@ class MessageReactionRemoveAll extends Action {
  * Emitted whenever all reactions are removed from a cached message.
  * @event Client#messageReactionRemoveAll
  * @param {Message} message The message the reactions were removed from
+ * @param {Collection} reactions The message reactions that were removed.
```

Comment:
Here generic types of the Collection can be specified, like the `messageDeleteBulk` event:
https://github.com/discordjs/discord.js/blob/fef19eecf7cebb9096203e0a4af2bc46e9ce1f68/src/client/actions/MessageDeleteBulk.js#L34
```suggestion
 * @param {Collection<Snowflake, MessageReaction>} reactions The message reactions that were removed.
```